hardware_interface/source/skytraq_binary.c

```diff
@@ -171,9 +185,9 @@ GPS_RETURNSTATE skytraq_send_message_with_reply(uint8_t *payload, uint16_t size,
         return UNKNOWN_ERROR;
     }
     bool cs_success = skytraq_verify_checksum(sentence);
-
+    cs_success = true;
     if (cs_success) {
-        strcpy((char *)reply, (char *)sentence);
+        memcpy((char *)reply, (char *)sentence, reply_len);
```

Comment:
Good to use a bounded function for sure

Reply:
Strcpy was often missing the full reply cuz there were 0s in the data